### PR TITLE
Changed Struct Variables names and Added two endpoints of "add & remo…

### PR DESCRIPTION
**Pull request by rushikesh-kulkarni**
…ve Service "

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -118,8 +118,8 @@ type AuthStub struct {
 	Secret  string `json:"secret" yaml:"secret"`
 }
 
-// Functions holds the config for the functions module
-type Functions struct {
+// ServicesModule holds the config for the service module
+type ServicesModule struct {
 	Services         Services `json:"services" yaml:"services"`
```
Comment (Jayesh333) [inline config/config.go]:
```suggestion
	Services         Services `json:"externalServices" yaml:"externalServices"`
```

File: utils/server/routes.go [hunk 2]
```diff
@@ -34,7 +34,8 @@ func (s *Server) routes(router *mux.Router, profiler bool, staticPath string) {
 	router.Methods("GET").Path("/v1/api/config/static").HandlerFunc(handlers.HandleLoadStaticConfig(s.adminMan, s.syncMan))
 	router.Methods("POST").Path("/v1/api/config/static").HandlerFunc(handlers.HandleStoreStaticConfig(s.adminMan, s.syncMan))
 	router.Methods("DELETE").Path("/v1/api/config/{project}").HandlerFunc(handlers.HandleDeleteProjectConfig(s.adminMan, s.syncMan, s.configFilePath))
-
+	router.Methods("POST").Path("/v1/api/config/{project}/{service}/add-service").HandlerFunc(handlers.HandleAddService(s.adminMan, s.syncMan))
+	router.Methods("DELETE").Path("/v1/api/config/{project}/{service}/remove-service").HandlerFunc(handlers.HandleDeleteService(s.adminMan, s.syncMan))
```
Comment (sharadregoti) [inline utils/server/routes.go]:
```suggestion
	router.Methods("DELETE").Path("/v1/config/{project}/services/{serviceName}").HandlerFunc(handlers.HandleDeleteService(s.adminMan, s.syncMan))
```

File: utils/handlers/config_services.go [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+		// Check if the request is authorised
+		if err := adminMan.IsTokenValid(token); err != nil {
+			w.WriteHeader(http.StatusUnauthorized)
+			json.NewEncoder(w).Encode(map[string]string{"error": err.Error()})
+			return
+		}
+
+		vars := mux.Vars(r)
+		service := vars["service"]
```
Comment (sharadregoti) [inline utils/handlers/config_services.go]:
```suggestion
		serviceName := vars["serviceName"]
```

File: utils/handlers/config_services.go [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+		}
+
+		w.WriteHeader(http.StatusOK) //http status codee
+		json.NewEncoder(w).Encode(map[string]interface{}{})
+		return
+
+	}
+}
+
+// HandleDeleteService is an endpoint handler which deletes a table in specified database
```
Comment (sharadregoti) [inline utils/handlers/config_services.go]: update the comment

File: utils/handlers/config_services.go [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+		conf, err := syncMan.GetConfig(project)
+		if err != nil {
+			w.WriteHeader(http.StatusInternalServerError)
+			json.NewEncoder(w).Encode(map[string]string{"error": err.Error()})
+			return
+		}
+
+		delete(conf.Modules.Services.Services, service)
+
+		syncMan.SetProjectConfig(conf)
```
Comment (sharadregoti) [inline utils/handlers/config_services.go]: handle error here

File: utils/handlers/config_services.go [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+
+		// Check if the request is authorised
+		if err := adminMan.IsTokenValid(token); err != nil {
+			w.WriteHeader(http.StatusUnauthorized)
+			json.NewEncoder(w).Encode(map[string]string{"error": err.Error()})
+			return
+		}
+
+		vars := mux.Vars(r)
+		service := vars["service"]
```
Comment (sharadregoti) [inline utils/handlers/config_services.go]:
```suggestion
		serviceName := vars["serviceName"]
```

File: utils/handlers/config_services.go [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+			json.NewEncoder(w).Encode(map[string]string{"error": err.Error()})
+			return
+		}
+
+		value := &config.Service{}
+		json.NewDecoder(r.Body).Decode(&value)
+
+		conf.Modules.Services.Services[service] = value
+
+		syncMan.SetProjectConfig(conf)
```
Comment (sharadregoti) [inline utils/handlers/config_services.go]: handle error here

**Comment by sharadregoti** [comment]
@rushikesh-kulkarni make the changes & push the code ASAP

**Pull request merged**